curriculum/challenges/portuguese/02-javascript-algorithms-and-data-structures/regular-expressions/match-numbers-and-letters-of-the-alphabet.portuguese.md

```diff
@@ -50,6 +50,8 @@ let result = myRegex; // Change this line
 <section id='solution'>
 
 ```js
-// solution required
+let quoteSample = "Blueberry 3.141592653s are delicious.";
+let myRegex = /[h-s2-6]/; // 
+let result =quoteSample.match(myRegex); //
```

Comment:
```suggestion
let quoteSample = "Blueberry 3.141592653s are delicious.";
let myRegex = /[h-s2-6]/; // Altere essa linha
let result =quoteSample.match(myRegex); // Altere essa linha
```
Can we add the comments to the solution to match the English version?

Reply:
@nhcarrigan Do not add any comments to the non-English curriculum files.   Any code comment that exists in the English file will be automatically translated when other languages are rolled out.  See PR #38040 for more information.

Reply:
The provided solution does not appear to pass the challenge tests - one test in particular requires the `g` and `i` flags.